README.md

```diff
@@ -23,30 +27,75 @@ It will compile the source code into an executable named `sonic`.
 | `Hash` | integer | $16$ | $[1, 1024]$ | Transposition table size (in MB). |
 | `Clearhash` | button | | | Clear entries in transposition table. |
 
-## Features
-
-- Search
-  - Alpha-Beta Search
-  - Quiescence Search
-  - Iterative Deepening
-  - Check Extension
-  - Transposition Table
-  - Repetition Detection
-  - Insufficient Mating Material
-  - Null Move Pruning
-  - Aspiration Window
-  - Principal Variation Search
-  - Mate Distance Pruning
-  - Futility Pruning
-  - Reverse Futility Pruning
-  - Late Move Reduction
-  - Delta Pruning
-- Move Ordering
-  - PV move
-  - Pawn Promotion
-  - MVV-LVA
-- Evaluation
-  - Piece Square Table
-  - Passed Pawn Bonus
-  - Piece Mobility
-  - Tapered Evaluation
+## ⚙️Features
+
+Sonic includes a variety of advanced chess engine features:
+
+### Search Algorithms
```

Comment:
How about a cute emoji for this section too! I would recommend 👓.

Reply:
> How about a cute emoji for this section too! I would recommend 👓.

check, i just did it.